component/admin/layouts/payment/logs_toolbar.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @package     Redshopb.Frontend
+ * @subpackage  Layouts
+ *
+ * @copyright   Copyright (C) 2012 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('_JEXEC') or die;
+
+$data = $displayData;
+
+$formName = $data['formName'];
+$return = isset($data['return']) ? $data['return'] : null;
+?>
+<h2>
+	<?php echo JText::_('COM_REDCORE_PAYMENT_LOG'); ?>
+</h2>
+<div class="container-fluid">
+	<div class="btn-toolbar toolbar">
+		<div class="btn-group">
+			<button class="btn"
+			        onclick="if (document.<?php echo $formName; ?>.boxchecked.value==0){alert('<?php echo JText::_('COM_REDCORE_PLEASE_SELECT_ITEM', true); ?>');}
+				        else{ Joomla.submitform('payment_log.edit', document.getElementById('<?php echo $formName; ?>'))}"
+			        href="#">
+				<i class="icon-edit"></i>
+				<?php echo JText::_('JTOOLBAR_EDIT') ?>
+			</button>
+			<button class="btn btn-danger"
+			        onclick="if (document.<?php echo $formName; ?>.boxchecked.value==0){alert('<?php echo JText::_('COM_REDCORE_PLEASE_SELECT_ITEM', true); ?>');}
+				        else{ Joomla.submitform('payment_logs.delete', document.getElementById('<?php echo $formName; ?>'))}"
+			        href="#">
+				<i class="icon-trash"></i>
+				<?php echo JText::_('JTOOLBAR_DELETE') ?>
```

Comment:
Log deletion might get a bit trickier in terms of ACL, since redCORE can be accessed by regular site administrators, so it might pose some security risk
I was thinking that the log deletion function should only change a "deleted" tag, and we could just allow super admins to empty the trash, to avoid that.